tests/license_check.py

```diff
@@ -26,6 +32,19 @@
     "GNU General Public License (GPL)",
     "Public Domain",
     "The Unlicense (Unlicense)",
+    "Apache License, Version 2.0",
+    "Apache License v2.0",
+    "Apache License 2.0",
+    "GNU Lesser General Public License v2 or later (LGPLv2+)",
```

Comment:
Which library has this license? @biobootloader are we ok with using this library?

Reply:
[chardlet](https://github.com/chardet/chardet) is the library.

Reply:
looks like it should be fine!